unit-tests/run-unit-tests.py

```diff
@@ -3,62 +3,71 @@
+sys.path = [p for p in sys.path if not file.is_inside(p, getusersitepackages())]
+
+
+# log.d( 'modified=', sys.path )
 
 
 def usage():
-    ourname = os.path.basename( sys.argv[0] )
-    print( 'Syntax: ' + ourname + ' [options] [dir]' )
-    print( '        dir: location of executable tests to run' )
-    print( 'Options:' )
-    print( '        --debug          Turn on debugging information (does not include LibRS debug logs; see --rslog)' )
-    print( '        -v, --verbose    Errors will dump the log to stdout' )
-    print( '        -q, --quiet      Suppress output; rely on exit status (0=no failures)' )
-    print( '        -s, --stdout     Do not redirect stdout to logs' )
-    print( '        -r, --regex      Run all tests whose name matches the following regular expression' )
-    print( '        -t, --tag        Run all tests with the following tag. If used multiple times runs all tests matching' )
-    print( '                         all tags. e.g. -t tag1 -t tag2 will run tests who have both tag1 and tag2' )
-    print( '                         tests automatically get tagged with \'exe\' or \'py\' and based on their location' )
-    print( '                         inside unit-tests/, e.g. unit-tests/func/test-hdr.py gets [func, py]' )
-    print( '        --list-tags      Print out all available tags. This option will not run any tests' )
-    print( '        --list-tests     Print out all available tests. This option will not run any tests' )
-    print( '                         If both list-tags and list-tests are specified each test will be printed along' )
-    print( '                         with its tags' )
-    print( '        --no-exceptions  Do not load the LibCI/exceptions.specs file' )
-    print( '        --context <>     The context to use for test configuration' )
-    print( '        --repeat <#>     Repeat each test <#> times' )
-    print( '        --config <>      Ignore test configurations; use the one provided' )
-    print( '        --no-reset       Do not try to reset any devices, with or without Acroname' )
-    print( '        --rslog          Enable LibRS logging (LOG_DEBUG etc.) to console in each test' )
+    ourname = os.path.basename(sys.argv[0])
+    print('Syntax: ' + ourname + ' [options] [dir]')
+    print('        dir: location of executable tests to run')
+    print('Options:')
+    print('        --debug          Turn on debugging information (does not include LibRS debug logs; see --rslog)')
+    print('        -v, --verbose    Errors will dump the log to stdout')
+    print('        -q, --quiet      Suppress output; rely on exit status (0=no failures)')
+    print('        -s, --stdout     Do not redirect stdout to logs')
+    print('        -r, --regex      Run all tests whose name matches the following regular expression')
+    print(
+        '        -t, --tag        Run all tests with the following tag. If used multiple times runs all tests matching')

```

Comment:
This line may have needed fixing, but not in the way that was done

Reply:
reverted to original file